stable/traefik/templates/configmap.yaml

```diff
@@ -77,3 +77,17 @@ data:
       {{- end}}
       {{- end}}
     {{- end }}
+    {{ if .Values.metrics.prometheus.enabled }}
+    [web.metrics.prometheus]
+      Buckets = {{ .Values.metrics.prometheus.buckets }}
+    {{- end }}
+    {{ if .Values.metrics.datadog.enabled }}
```

Comment:
The [intent](https://github.com/containous/traefik/blob/69c628b626a01e40dda33368534e1b8a59791bda/server/server.go#L1069) of the author was to have traefik support multiple metric collection tools.